sync_radar_data_globus.py

```diff
@@ -120,14 +113,16 @@ def __init__(self, client_id, client_secret=None, transfer_rt=None):
         parser.add_argument("-t", "--data_type",
                             help="One of {} Default: 'raw'".format(self.possible_data_types),
                             default='raw')
+
         parser.add_argument("sync_local_dir", help="Path on endpoint to sync data to")
         args = parser.parse_args()
```

Comment:
huh? didn't know you could use parse_args this way, I get it to return a dictionary for me lol

Reply:
Interesting. I always do the parse_args() way too.